Netskope/netskope-transaction/ingest/parser.yml

```
@@ -0,0 +1,243 @@
+name: netskope-transaction
+ignored_values: ["-", "NotAvailable", "N/A"]
```

I wonder if you cannot add `NoSSL` and `NotChecked` also in the ignored values
```suggestion
ignored_values: ["-", "NotAvailable", "N/A", "NoSSL", "NotChecked"]
```